assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -66,6 +66,7 @@ export const DropDownSelect = styled.button`
   border-radius: 4px;
   padding: 2px 0 2px 6px;
   cursor: pointer;
+  width: fit-content;
```

Comment:
Does this change all drop downs? Are we sure we want to do that?

Reply:
Good catch! Added a conditional check to handle the issue.

Find a good case for testing:
Before the fix:
![before](https://user-images.githubusercontent.com/11252886/103378937-ddb45700-4a98-11eb-9477-0e2fd01612fb.png)

After the fix:
![after](https://user-images.githubusercontent.com/11252886/103378948-e60c9200-4a98-11eb-8cc9-22486a2ba1da.png)